ee/agent/knapsack/knapsack.go

```diff
@@ -92,6 +92,11 @@ func (k *knapsack) SetInstanceQuerier(q types.InstanceQuerier) {
 	k.querier = q
 }
 
+// RegistrationTracker interface methods
+func (k *knapsack) RegistrationIds() []string {
```

Comment:
no strong preference here but do we want to use `RegistrationIDs` for consistency with e.g. `GetRunID`?

Reply:
Done!